syscall/fdtable.c

```diff
@@ -283,6 +283,9 @@ int oe_fdtable_reassign(int fd, oe_fd_t* new_desc, oe_fd_t** old_desc)
     int ret = -1;
     bool locked = false;
 
+    if (!new_desc || !old_desc)
+        OE_RAISE_ERRNO(OE_EINVAL);
+
 #if !defined(NDEBUG)
     _assert_fd(new_desc);
 #endif
```

Comment:
Line 293 can be removed since 286-287 mean you can't get here without the condition being true